Updating env page (#8811)
fishtown-analytics/docs.getdbt.com: website/docs/docs/cloud/about-profiles.md

Assign profiles to [deployment environments](/docs/dbt-cloud-environments#deployment-environment) and reuse those profiles in other deployment environments within the same project. You can manage profiles programmatically using our [API documentation](/dbt-cloud/api-v3#/operations/List%20Profiles).

### Environment profiles table

On an environment's **Settings** page, the **Connection profiles** section lists the profiles assigned to that environment:

> - **Profile name**: Click a profile name to open the view/edit drawer (view or edit mode). In view mode there is no separate action column; use the profile name to open and view a profile.

Reviewer: ```suggestion
- **Profile name**: Click a profile name to open the view/edit drawer. In view mode, there’s no separate action column, so use the profile name to open and view a profile.
```

Reviewer: Hiya @runleonarun 

Just made a small tweak here:
- removed (view or edit mode) as it seemed a bit redundant after view/edit drawer
- tightened the sentence a tad

1. From the main menu, click **Orchestration** and select **Environments**.
2. Click an available deployment environment.
3. Click **Settings**, then click **Edit**.
4. Navigate to the **Connection profiles** section and click **Edit**.

> 5. Click the **swap icon** (change profile) next to an existing profile row, then select **Create new profile** from the **Profile** dropdown.

Reviewer: Hiya @runleonarun 

I think step 5 could be amended a bit. When I followed the steps in my instance - I wasn't immediately aware of where the swap icon was.

> 5. Click the **swap icon** (change profile) next to an existing profile row, then select **Create new profile** from the **Profile** dropdown.

Reviewer: ```suggestion
5. Click the **swap icon** (change profile) in the **Connection profiles** row, then select **Create new profile** from the **Profile** dropdown.
```

Reviewer: Or maybe add a pic / carousel highlighting the swap icon.
